spec/rails_helper.rb

```diff
@@ -678,10 +678,9 @@ def using_session_with_localhost_resolution(name, &block)
       system_tests_initialized = true
     end
 
-    driver = [:selenium]
+    driver = [:playwright]
     driver << :mobile if example.metadata[:mobile]
     driver << :chrome
-    driver << :headless unless ENV["SELENIUM_HEADLESS"] == "0"
```

Comment:
I removed this, but could eventually remove it if necessary. Now the driver will just follow the ENV var